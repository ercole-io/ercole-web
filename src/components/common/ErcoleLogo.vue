<template>
  <img src="@/assets/images/scritta-bianca-cane-blu-copia.png" alt="Ercole" />
</template>

<script>
export default {
  name: 'commom-ercolelogo-component',
}
</script>

<style lang="scss" scoped></style>
