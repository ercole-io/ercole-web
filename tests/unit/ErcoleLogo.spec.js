import ErcoleLogo from '@/components/common/ErcoleLogo'
import { mount } from '@vue/test-utils'

describe('Ercole Logo', () => {
  it('will mount the component', () => {
    const wrapper = mount(ErcoleLogo)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
