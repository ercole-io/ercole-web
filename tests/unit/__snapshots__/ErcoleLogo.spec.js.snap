// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ercole Logo will mount the component 1`] = `<img src="@/assets/images/ercole-logo-white.svg" alt="Ercole">`;
